packages/data/src/reviews/actions.js

```diff
@@ -12,6 +19,60 @@ export function updateReviews( query, reviews, totalCount ) {
 	};
 }
 
+export function* updateReview( reviewId, reviewFields, query ) {
```

Comment:
What do you think about creating some status-specific actions like `markReviewAsSpam()` and `approveReview()` so that the component code can be a little simpler?

Reply:
Yeah not a bad idea, the only caveat was 'reverting' a change, which made having a generic `update` function rather handy. I could do both. In either case would it make sense to move the `createNotice` to the action as well?

Reply:
Ah right.. the previous status won't always be the same making reversal unable to take advantage of the "simpler" actions. Let's scrap the idea 🙃 .

> In either case would it make sense to move the `createNotice` to the action as well?

No I don't think we'd want to couple those in the event that UX needed to change or the action could be used in multiple different contexts.